RELEASES.md

```diff
@@ -8,6 +8,13 @@
 - Added `InputMap::Clear`.
 - Fixed [a bug](https://github.com/Leafwing-Studios/leafwing-input-manager/issues/430) related to incorrect axis data in `Chord` when not all buttons are pressed.
 
+### Enhancements
+
+- Changed `DualAxis` and `SingleAxis` deadzones to be more accessible. 

```

Comment:
"More accessible" is really unclear to the reader.

Reply:
That's true. I could remove the that bullet point and just list the points underneath? I hoping that the following bullets would explain it, but I understand how that can be unclear.

Reply:
```suggestion
- Improved deadzone handling for both `DualAxis` and `SingleAxis` deadzones 
```